smartsim/_core/mli/infrastructure/storage/dragonfeaturestore.py

```diff
@@ -69,3 +71,23 @@ def __contains__(self, key: str) -> bool:
         Return `True` if the key is found, `False` otherwise
         :param key: Unique key of an item to retrieve from the feature store"""
         return key in self._storage
+
+    @property
+    def descriptor(self) -> str:
+        """Return a unique identifier enabling a client to connect to
+        the feature store
+        :returns: A descriptor encoded as a string"""
+        return str(self._storage.serialize())
+
+    @classmethod
+    def from_descriptor(
+        cls,
+        descriptor: str,
+        # b64encoded: bool = False,
```

Comment:
removable comment

Reply:
i don't know what you're referring to! 👻